docs/source/getting_started.rst

```diff
@@ -21,6 +21,15 @@ Alternatively, add the following line to your project's
 
    djangocms-frontend
 
+**djangocms-frontend** has weak dependencies you can install separately or
+by adding an option:
+
+.. code::
+
+    djangocms-frontend[reCaptcha]  # Installs django-recaptcha enabling reCaptcha for forms
+    djangocms-frontend[djangocms-icon]  # Installs djangocms-icon for icons support in links
+    djangocms-frontend[all]  # Installs all of the above
```

Comment:
You can comma separate the extra packages in a pip file. I think that explicit approach would better than an "all" which isn't clear what it'll come with.

Reply:
Removed now.